lib/client/dispatcher/client/api.ts

```diff
@@ -20,35 +20,47 @@ export class HttpDispatcherServiceClient implements DispatcherServiceClient {
     params: CreateConnectionRequestParams,
     data: CreateConnectionRequestData,
   ): Promise<ServerId> {
-    const path = `/hidden/broker/${params.hashedBrokerToken}/connections/${params.brokerClientId}`;
+    try {
+      const path = `/hidden/broker/${params.hashedBrokerToken}/connections/${params.brokerClientId}`;
 
-    const response = await axiosInstance.post<CreateConnectionResponse>(
-      path,
-      {
-        data: {
-          attributes: {
-            deployment_location: data.deployment_location,
+      const response = await axiosInstance.post<CreateConnectionResponse>(
+        path,
+        {
+          data: {
+            attributes: {
+              deployment_location: data.deployment_location,
+            },
           },
         },
-      },
-      {
-        baseURL: this.baseUrl,
-        headers: {
-          'Content-type': 'application/vnd.api+json',
+        {
+          baseURL: this.baseUrl,
+          headers: {
+            'Content-type': 'application/vnd.api+json',
+          },
+          params: { version: this.version },
+          validateStatus: () => true,
         },
-        params: { version: this.version },
-        validateStatus: () => true,
-      },
-    );
+      );
 
-    const apiResponse = response?.data?.data;
-    const snykRequestId = response.headers['snyk-request-id'] || '';
-    logger.trace(
-      { snykRequestId, apiResponse },
-      'createConnection API response',
-    );
+      const apiResponse = response.data.data;
+      if (!apiResponse?.attributes) {
+        logger.trace(
+          { apiResponse },
+          'Unexpected Connection Allocation Server response',
+        );
+        throw new Error('Unexpected connection cllocation server response');
```

Comment:
please fix this typo

Reply:
done with 1948bfe55cf671d0fc1fa268b6250fbc0e096575